org.osgi.service.jakartaws/src/org/osgi/service/jakartaws/whiteboard/WebserviceWhiteboardConstants.java

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0 
+ *******************************************************************************/
+package org.osgi.service.jakartaws.whiteboard;
+
+/**
+ * Defines standard constants for the Jakarta Web Services Whiteboard services.
+ * 
+ * @author $Id$
+ */
+public class WebserviceWhiteboardConstants {
+
+    private WebserviceWhiteboardConstants() {
+        // non-instantiable
+    }
+
+	/**
+	 * Base namespace for the Webservice Whiteboard specification
+	 */
+	public static final String	WEBSERVICE							= "osgi.ws";
```

Comment:
osgi.ws
osgi.jakartaws
osgi.jakartaxmlws
osgi.jakarta.ws
oagi.jakarta.xml.ws

@timothyjward 
@laeubi is this in line with what we jave in the other jakarta specs?

Reply:
https://github.com/osgi/osgi/blob/b32032ea3bb23de7d3b27622dc3865dfc803cef5/org.osgi.service.jakartars/src/org/osgi/service/jakartars/whiteboard/JakartarsWhiteboardConstants.java#L220

Reply:
osgi.jakarta.xml.ws